modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -1078,6 +1108,33 @@ private void _checkObjectEntryObjectDefinitionId(
 		}
 	}
 
+	private ObjectEntry _copyVersionedObjectEntry(
+			DTOConverterContext dtoConverterContext, ObjectEntry objectEntry,
+			ObjectDefinition objectDefinition)
+		throws Exception {
+
+		Map<String, Object> properties = objectEntry.getProperties();
+
+		List<ObjectField> objectFields =
+			_objectFieldLocalService.getCustomObjectFields(
+				objectDefinition.getObjectDefinitionId());
+
+		for (ObjectField objectField : objectFields) {
```

Comment:
Can you inline `objectFields`?

Reply:
The SF does not allow this, other places like [this one](https://github.com/liferay/liferay-portal/blob/b85c01ef7de52fad56aa7e4e40945c13e55562ab/modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/dto/v1_0/converter/ObjectEntryDTOConverter.java#L1075-L1079) also do the same thing due to the same reason.